### ISSUE TITLE
Add suboption "recurse into subdirectories" to scout

### ISSUE THREAD

**Issue by pavel-the-best** [ranger/ranger#2066]
This would allow implementing recursive search without python extensions.

**Comment by chu4ng** [comment]
I think we can use flat to do this.
Otherwise scout need depth.

**Comment by toonn** [comment]
Yes, `:flat` is the closest thing in ranger. The wiki has integrations that can do better with ripgrep and silversearcher and other commands.
